src/select.coffee

```diff
@@ -400,6 +400,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
     )
   .property 'groupedContent.[]'
 
+  # whether or not to ensure that highlighted option is visible
```

Comment:
What? Rename the function too. This name is super weird. Even just reading the description for the PR, I had no idea what this change was doing.